code_challenges/code_wars/remove_parentheses.py

```diff
@@ -0,0 +1,36 @@
+class CharacterRemover:
+    def __init__(self, characters):
+        self.characters = characters
+
+    def item_to_remove(self):
+        self.start_item = "("
+        start_item_index = None
+        self.stop_item = ")"
+        stop_item_index = None
```

Comment:
While Python will happily interpret this, it's generally considered best practice to define all properties that will exist in the init block even if they start out without a value.

Anything that you will attach to `self`.  So in this case `self.start_item` and  `self.stop_item`.  There are probably exceptions to this rough philosophy, but especially if the values don't change throughout the life of this instance, they should be present on init.  So since start is always `(` and end `)` moving them into the init doesn't change the way the code operates at all and brings this inline with what most would expect.